server/src/main/java/io/druid/server/coordination/DruidServerMetadata.java

```diff
@@ -27,7 +27,8 @@
 public class DruidServerMetadata
 {
   private final String name;
-  private final String host;
+  private final String hostAndPort;
+  private final String hostAndTlsPort;
```

Comment:
This is used for announcing on ZK. So, wanted to keep announcement similar and just add one more field. I thought it might be easier to keep backward compatibility and less changes in code elsewhere. I don't know, if it is really required then I may take a shot at changing it.